nsfc-plugin/pom.xml

```diff
@@ -0,0 +1,657 @@
+<!--
+    Copyright (c) Intel Corporation
+    Copyright (c) 2017
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+	<parent>
+	    <groupId>org.osc.plugin</groupId>
+		<artifactId>sdn-controller-nsfc-plugin</artifactId>
+		<version>2.0.1-SNAPSHOT</version>
+	</parent>
+
+    <groupId>org.osc.plugin</groupId>
+    <artifactId>nsfc-plugin</artifactId>
+    <name>org.osc.plugin:nsfc-plugin</name>
+
+    <properties>
```

Comment:
please revist the entire file. group dependencies by project, scope etc. remove unused dependencies